app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -102,7 +102,11 @@ def self.raw_connect(host, port, username, password, security_protocol, cert_sto
       options = {
         :tenant         => 'hawkular',
         :verify_ssl     => verify_ssl_mode(security_protocol),
-        :ssl_cert_store => cert_store
+        :ssl_cert_store => if cert_store.kind_of?(String)
```

Comment:
Functional-wise I'm glad the change is that simple, though stylistically I would prefer a bit different way, something like:
```
if cert_store.kind_of?(String)
  cert_store = Endpoint.new(:certificate_authority => cert_store).ssl_cert_store

options = {
  ...
  :ssl_cert_store => cert_store
  ...
}
```

Reply:
I prefer cert_store.kind_of?(String) ? Endpoint.new(:certificate_authority => cert_store).ssl_cert_store : cert_store ^^ Vote community !!!

Reply:
Ternary operators before if

Reply:
@tumido wins...I'll change the code ^^ thanks